packages/pundle/examples/Full Example (.pundle.js).js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  debug: true,
+  // ^ Setting this to true sets "process.env.NODE_ENV" to "development" in processed js, it's set to "production" otherwise
+  entry: ['./index.js'],
+  output: {
+    bundlePath: '/bundle.js',
+    sourceMap: true,
+    sourceMapPath: '/bundle.js.map',
+  },
+  server: {
+    port: 8090,
+    hmrPath: '/__sb_pundle_shiny_hmr',
+    bundlePath: '/bundle.js',
+    sourceMapPath: '/bundle.js.map',
+    redirectNotFoundToIndex: true,
+  },
+  presets: [
+    [require.resolve('pundle-preset-default'), {
+      resolver: { extensions: ['', '.js', '.json', '.ts', '.tsx'] },
+    }],
+  ],
```

Comment:
YES!  So glad to have an example.

You should add the TypeScript plugin a) to show where plugins go and b) because you used `.tsx` in the example, but don't have a way to interpret those files.